images/win/scripts/Installers/Install-MSYS2.ps1

```diff
@@ -0,0 +1,92 @@
+################################################################################
+##  File:  Install-MSYS2.ps1
+##  Desc:  Install MSYS2 and 64-bit gcc, cmake, & llvm (32-bit commented out)
+################################################################################
+
+# References
+# https://github.com/msys2/MINGW-packages/blob/master/azure-pipelines.yml
+# https://packages.msys2.org/group/
+
+$dash = "—"
+$wid = 80
+
+$orig_path = $env:PATH
+$git_path  = "C:\Program Files\Git"
```

Comment:
We have the environment variable `AGENT_TEMPDIRECTORY` in the image with the path to temp files/folders. Can we use it?